# DP: Fixes some old compatibility issues with m68k and lchown/chown

--- ports/sysdeps/unix/sysv/linux/m68k/chown.c	2006-01-08 09:21:17.000000000 +0100
+++ ports/sysdeps/unix/sysv/linux/m68k/chown.c	2006-06-01 11:36:57.000000000 +0200
@@ -67,3 +67,8 @@
 }
 libc_hidden_def (__chown)
 weak_alias (__chown, chown)
+
+#include <shlib-compat.h>
+#if SHLIB_COMPAT (libc, GLIBC_2_1, GLIBC_2_2)
+compat_symbol (libc, __chown, chown, GLIBC_2_1);
+#endif
--- ports/sysdeps/unix/sysv/linux/m68k/lchown.c	2000-01-17 06:20:35.000000000 +0100
+++ ports/sysdeps/unix/sysv/linux/m68k/lchown.c	2006-06-01 11:36:57.000000000 +0200
@@ -1 +1,6 @@
 #include <sysdeps/unix/sysv/linux/i386/lchown.c>
+
+#include <shlib-compat.h>
+#if SHLIB_COMPAT (libc, GLIBC_2_1, GLIBC_2_2)
+compat_symbol (libc, __lchown, lchown, GLIBC_2_1);
+#endif
